src/EnergyPlus/DXCoils.cc

```diff
@@ -14658,6 +14658,7 @@ void ReportDXCoil(EnergyPlusData &state, int const DXCoilNum) // number of the c
     // Using/Aliasing
     auto &DXElecCoolingPower = state.dataHVACGlobal->DXElecCoolingPower;
     auto &DXElecHeatingPower = state.dataHVACGlobal->DXElecHeatingPower;
+    auto &DefrostElecPower = state.dataHVACGlobal->DefrostElecPower;
     auto &TimeStepSys = state.dataHVACGlobal->TimeStepSys;
```

Comment:
All of these are scalars and should be removed and replaced in code with state.dataHVACGlobal->*

Reply:
Also, don't reference TimeStepSys, just use Real64 TimeStepSys = ...

Reply:
OK. Will change them. These are in existing code.

Reply:
Done, also applied ShortCuts to `state.dataDXCoils->DXCoil(DXCoilNum)`

Reply:
> OK. Will change them. These are in existing code.

Yes, they are all over the existing code and we need to sweep and get them out. They are actively making the code both less readable and slower. FWIW in the parlance these sorts of things are called "sloptimizations".